css/desktop.css

```diff
@@ -109,23 +109,56 @@ header {
 
 .section3-container {
   .wrapper {
-    display: grid;
-    grid-template-columns: repeat(4, 1fr);
+    .link-wrapper {
+      display: grid;
+      grid-template-columns: repeat(4, 1fr);
 
-    row-gap: 40px;
-    column-gap: 20px;
+      row-gap: 40px;
+      column-gap: 20px;
 
-    .link-container {
-      .link-content {
-        img {
-          z-index: -1;
-        }
+      .link-container {
+        position: relative;
 
-        span {
-          z-index: 1;
+        .link-content {
+          img {
```

Comment:
why useing type? why you don't using class name?

Reply:
I used img because it's simple icon image on the grid, I'll put class if it's necessary